etl/steps/data/grapher/agriculture/2023-04-21/uk_long_term_yields.py

```diff
@@ -0,0 +1,42 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from owid.catalog import Dataset
+
+from etl.helpers import PathFinder, create_dataset, grapher_checks
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load garden dataset.
+    ds_garden: Dataset = paths.load_dependency("uk_long_term_yields")
+
+    # For simplicity, remove sources descriptions (which are mostly irrelevant for the variable used here).
+    ds_garden.metadata.sources[0].description = ""
+    ds_garden.metadata.sources[1].description = ""
+    ds_garden.metadata.sources[2].description = ""
```

Comment:
Why do you remove the description of the sources?

Reply:
This is not a great solution, but the faostat sources description is very long and mentions things that are unrelated to the relevant variables. That's why I manually removed it.